README.md

```diff
@@ -1 +1,25 @@
-# mysqld_exporter (UNDER CONSTRUCTION)
+# mysqld_exporter
+
+Exporter for MySQL server metrics http://prometheus.io/
+
+## Building and running
+
+    make
+    ./mysqld_exporter <flags>
+
+## Configuration
+The configuration is in JSON. An example with all possible options:
```

Comment:
IMHO this is bad idea show login/password in process list.
But if you think this is needed - I will.